flask/app/templates/stats.html

```diff
@@ -9,9 +9,9 @@ <h1 style="background-color:transparent; margin-top: -175px; padding: 0px;">Play
         <img src="./static/img/female.png" alt="Avatar1" style="border-radius:50%; height: 300px; width: 300px;">
         <h2>You</h2>
         <div style="text-align: left; margin-left: 20%;">
-          <p>Wins: #</p>
-          <p>Losses: #</p>
-          <p>Ties: #</p>
+          <p>Wins: <span id="my-wins"></span></p>
```

Comment:
I have been using "player" all over, so lets keep it consistent and use "player-wins" etc.

Reply:
fixed